client/components/app/partials/noData.html

```diff
@@ -4,40 +4,15 @@
     <h2>{{i18n 'app.configure_your_app'}}</h2>
 
     <p>1. {{i18n 'app.config_install_pkg'}}</p>
-    <pre><code>meteor add meteorhacks:kadira
-
-  // for Meteor 0.8.3 and older versions
-  mrt add kadira</code></pre>
+    <pre><code>meteor add lmachens:kadira</code></pre>
 
     <p>2. {{{i18n 'app.config_connect_code'}}}</p>
     <div id="no-data-code-wrap">
-      <ul class="nav nav-tabs" role="tablist">
```

Comment:
Actually I like to have the option to set the apm settings via code or env. So I would not remove these tabs here and better add the apm engine url to the other tabs too:
Code:
`Kadira.connect('{{state$appId}}', '{{state$secret}}', { endpoint:  apmEngineUrl });`
For env:
KADIRA_OPTIONS_ENDPOINT (https://github.com/lmachens/kadira/blob/master/lib/environment_variables.js#L67)